chain/indexer/README.md

```diff
@@ -70,3 +67,35 @@ $ cargo run --release ~/.near/betanet
 ```
 
 After the network is synced, you should see logs of every block produced in Betanet. Get back to the code to implement any custom handling of the data flowing into the indexer.
+
+## Tweaks
+
+By default, nearcore is configured to do as little work as possible while still operating on an up-to-date state. Indexers may have different requirements, so there is no solution that would work for everyone, and thus we are going to provide you with the set of knobs you can tune for your requirements.
+
+As already has been mentioned in this README, the most common tweak you need to apply is listing all the shards you want to index data from; to do that, you should ensure that `"tracked_shards"` in the `config.json` lists all the shard IDs, e.g. for the current betanet and testnet, which have a single shard:
```

Comment:
I think tracking shards is necessary for indexer, unless you only want to index block data (which doesn't seem to be super useful).

Reply:
Is my wording not clear enough? Should I stress it out more? (it is already mentioned 3 times in this single README, and we also have an assertion in the implementation :smile:)

Reply:
Never mind. I think I misread.